packages/mdc-elevation/package.json

```diff
@@ -14,6 +14,6 @@
   },
   "dependencies": {
     "@material/animation": "^0.34.0",
-    "@material/theme": "^0.4.0"
+    "@material/theme": "^0.35.0"
```

Comment:
Is 0.35.0 the most recent version of the theme packages?

Reply:
[Yes](https://github.com/material-components/material-components-web/blob/v0.36.0/packages/mdc-theme/package.json#L4)